docs/test_runner.md

```diff
@@ -0,0 +1,163 @@
+En este documento, presentaré tanto los criterios para la elección del test runner y de la biblioteca de aserciones como una breve exposición de los test runners y las bibliotecas de aserciones a tener en cuenta. Finalmente presentaré las herramientas elegidas junto al porque de dicha elección.
+
+# TEST RUNNER
+
+El siguiente paso para avanzar en mi proyecto es elegir el test runner que mejor se adapte a este. Para ello, voy a tener en cuenta los siguientes criterios:
+
+1. **Actualizado:** el test runner debe tener actualizaciones frecuentes. El hecho de que tenga versiones actualizadas nos da la seguridad de que puede estar adaptado a las nuevas tecnologías (nos evitaremos de este modo usar un test que haya quedado obsoleto). Además si el test está actualizado, se han podido solucionar los errores que pueda tener.
+2. **Velocidad:** El tiempo que se necesita para ejecutar un conjunto de pruebas es importante a la hora de decidir sobre el  uso de un test runner u otro. Encontrar los errores lo antes posible es de ayuda a los desarrolladores y ahorrará tiempo de cómputo.
+3. **Compatibilidad:** Es otro criterio importante, ya que buscamos un test runner que sea compatible con el lenguaje que estoy utilizando en el proyecto (en este caso, node.js). Además sería conveniente que el test fuese compatible con varios navegadores.
+4. **Uso por la comunidad:** El que un test runner sea utilizado puede indicar que es útil para la comunidad y que su funcionamiento es correcto. En este sentido, el que una herramienta tenga muchas descargas podríamos considerarlo como un indicativo de su uso.
+
+La elección del test runner esta basada en la información extraída de varias páginas (entre las que destaco las expuestas al final del documento). A partir de dicha información procedo a exponer los test runners más relevantes compatibles con JavaScript:
+
+
+### mocha
+
+**VENTAJAS**
+
+- Es un testing framework de JavaScript destinado a probar aplicaciones que se ejecutan con Node.js.
+- Es de código abierto y proporciona flexibilidad.
```

Comment:
¿Flexibilidad en qué? De verdad, ¿podéis intentar ceñiros a los criterios *que vosotros mismos habéis establecido*?